apps/studio/src/components/Datatable/DatatablePagination.tsx

```diff
@@ -0,0 +1,28 @@
+import { Pagination } from "@opengovsg/design-system-react"
+import { type Table } from "@tanstack/react-table"
+
+export interface DataTablePaginationProps<D> {
+  instance: Table<D>
+  pageSizeOptions?: number[]
```

Comment:
nit: should we remove this? doesn't seem to be used

Reply:
thx for the catch. removed.